subprojects/docs/src/docs/release/notes.md

```diff
@@ -54,6 +54,12 @@ In this release, plugin authors can use the `ObjectFactory.fileCollection()` met
 
 [The JaCoCo plugin](userguide/jacoco_plugin.html) has been upgraded to use [JaCoCo version 0.8.3](http://www.jacoco.org/jacoco/trunk/doc/changes.html) instead of 0.8.2 by default.
 
+## Better help message on delete operation failure
+
+The `:clean` task, all `Delete` tasks, and `project.delete {}` operations now provide with a better help message when failing to delete files. The most frequent and hard to troubleshoot causes for failing to delete files are other processes holding file descriptors open, and, concurrent writes.
```

Comment:
"...operations now provide a better help message..." (remove "with")
"...descriptors open, and concurrent writes." (remove extra comma)